internal/sign/manager.go

```diff
@@ -3,10 +3,70 @@ package sign
 import (
 	"context"
 
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
 	kmmv1beta1 "github.com/kubernetes-sigs/kernel-module-management/api/v1beta1"
 	"github.com/kubernetes-sigs/kernel-module-management/internal/utils"
 )
 
+// SyncOptions configures optional arguments of the Sync call.
+type SyncOptions struct {
+	// TargetImage configures the signed image to be pushed. When left
+	// empty no image is pushed, the signed files are just listed.
+	TargetImage string
```

Comment:
The same comment as for the build: WDYT about rename it and making it boolean?

Reply:
See https://github.com/kubernetes-sigs/kernel-module-management/pull/156#discussion_r1037104809

Reply:
ok